rules/concepts.md

```diff
@@ -48,6 +48,151 @@ In some cases, it is useful to have a rule that is only manually triggered.
+6. Day of Week
+7. Year (optional field)
+
+You may use the generator at [FreeFormatter.com](https://www.freeformatter.com/cron-expression-generator-quartz.html) or the openHAB WebUI rule setup to generate your cron expressions.
+
+### **Channel**-based Triggers
+
+Some add-ons provide trigger channels.
+Compared with other types of channels, a trigger channel provides information about discrete events, but does not provide continuous state information.
+
+Your rules can take actions based upon trigger events generated by these trigger channels.
+You can decide whether you want to catch only a specific or any trigger the channel provides.
+
+When a binding provides such channels, you can find the needed information in the corresponding binding documentation.
+There is no generic list of possible trigger events.
+The `triggerEvent`(s) available depend upon the specific implementation details of the binding.
+
+If the rule needs to know what the received event or the triggering channel was, use the [Available Values](#available-values).
+
+### **Thing**-based Triggers
+
+Your rules can take actions based upon status updates or status changes generated by Things.
+You can decide whether you want to catch only a specific or any status the Thing can get updated too.
+
+You can find all the possible values for status from [Thing Status](/docs/concepts/things.html).
+
+The `thingUID` is the identifier assigned to the Thing, manually in your configuration or automatically during auto discovery.
+You can find it from UI or from `.things` files.
+For example, one z-wave device can be "zwave:device:c5155aa4:node14".
+
+If the rule needs to know what the triggering Thing was, or access the previous or new status, use the [Available Values](#available-values).
+
+<!-- TODO: Update reference when actions are included in the reworked rule docs. -->
+Refer to [Thing Status Action](/docs/configuration/actions.html#thing-status-action) to find how to get the new thing status details or description in the script.
+
+### **System**-based Triggers
+
+System-based triggers are provided as described in the table below:
+
+| Trigger                            | Description                                                                                                                                                                                                                                          |
+|------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| System started                     | `System started` is triggered upon openHAB startup. In openHAB version 2, `System started` is also triggered after the rule file containing the System started trigger is modified, or after item(s) are modified in a .items file.                  |
```

Comment:
Let's not talk about OH 2 here. As time goes on, there will be fewer and fewer people reading these docs that have ever used OH 2. In OH 3, I believe `System started` corresponds with reaching start level 40 but we need to verify that. I know it's earlier than one would expect.

Reply:
Okay, I‘ll remove that OH 2 stuff. This comes, because I copied most parts from the Rules DSL page.

`System started` could correspond with start level 40, as openHAB is running then, but the startup is not completed yet. In fact that doesn‘t really matter for us.